shared/lib/ui-utils.js

```diff
@@ -22,3 +22,6 @@ export const TRANSACTION_SIMULATIONS_LEARN_MORE_LINK =
 
 export const GAS_FEES_LEARN_MORE_URL =
   'https://community.metamask.io/t/what-is-gas-why-do-transactions-take-so-long/3172';
+
+export const SMART_ACCOUNT_INFO_LINK =
```

Comment:
I think we could use ZENDESK_URLS.ACCOUNT_UPGRADE instead of creating a new constant here

ref: ui/helpers/constants/zendesk-url.js

also, this link is not working for me. When is the link scheduled to work? 
![CleanShot 2025-04-09 at 12 40 22](https://github.com/user-attachments/assets/b8cf2b43-d515-4039-ac45-acc2f408571e)

Reply:
Sure, I will make changes in next [PR](https://github.com/MetaMask/metamask-extension/pull/31774) to move this one faster.

Reply:
Addressed in [PR](https://github.com/MetaMask/metamask-extension/pull/31774)